ui/00_message/pawn/p0001.xml

```diff
@@ -5,10 +5,12 @@
   </message>
   <message>
     <original>我らはいつも、覚者さまとともに</original>
+	<translation>The Rift is always with us.</translation>
   </message>
   <message>
     <original>覚者さま、お力添えの機会
 お待ちしています</original>
+	<translation>Master, we look forward in assisting you.</translation>
```

Comment:
Small nitpick, as most of my comments will likely be, but I believe the phrase should be "look forward to".